data_diff/databases/databricks.py

```diff
@@ -84,14 +80,21 @@ def query_table_schema(self, path: DbPath) -> Dict[str, tuple]:
         # https://docs.databricks.com/spark/latest/spark-sql/language-manual/information-schema/columns.html
         # So, to obtain information about schema, we should use another approach.
 
+        conn = self.create_connection()
```

Comment:
Why create a separate connection for querying the schema?

Reply:
It is because of our threaded database architecture. For that cases we execute sql code via query workers. However, for databricks information about schema is downloaded not via sql queries but using specific python databricks client, and to obtain the schema, we should already have a connection in this place.

Reply:
I can write a method to call in in query workers but i think it is easier to just create a connection here because workers just wraps these connections themselves.